tests/models/test_hparams.py

```diff
@@ -580,8 +562,7 @@ class SuperClassPositionalArgs(BoringModel):
 
     def __init__(self, hparams):
         super().__init__()
-        self._hparams = None  # pretend BoringModel did not call self.save_hyperparameters()
```

Comment:
is this comment not meaningful anymore?
I put it there to avoid anyone deleting the line.
This was part of a regression test for a bugfix.

Reply:
I'll put it back